cumulus/primitives/storage-weight-reclaim/src/lib.rs

```diff
@@ -256,6 +256,10 @@ mod tests {
 		});
 	}
 
+	fn get_current_storage_weight() -> PerDispatchClass<Weight> {
```

Comment:
```suggestion
	fn get_current_weight() -> PerDispatchClass<Weight> {
```

This gets all the weight and not just storage. Should be used in the other tests as well.

Reply:
done